modules/library.js

```diff
@@ -0,0 +1,113 @@
+import Book from './book.js';
+import StrShelf from './strshelf.js';
+
+export default class Library {
+  shelf = [];
+
+  bookshelf;
+
+  addButton = document.querySelector('.add-btn');
+
+  newTitle = document.getElementById('newTitle');
+
+  newAuthor = document.getElementById('newAuthor');
+
+  frontShelf = document.getElementById('frontShelf');
+
+  removeButton;
+
+  static pushToStorage = (obj) => {
+    const stringify = JSON.stringify(obj);
+    localStorage.setItem('strShelf', stringify);
+  }
+
+  static displayBook = (title, author, id) => `
+      <span class="title-author">"${title}" by ${author}</span>
+      <button class="removeButton" id="${id}" type="button">Remove</button>
+      `;
+
+  pullFromStorage = () => {
+    const parsed = JSON.parse(localStorage.getItem('strShelf'));
+    let counter = this.shelf.length;
+    const preShelf = [];
+    for (let i = 0; i < this.shelf.length; i += 1) {
+      const parsedBook = parsed[`${counter}`];
+      const tempShelf = Library.displayBook(parsedBook.title, parsedBook.author, counter);
+      preShelf.unshift(tempShelf);
+      counter -= 1;
+    }
```

Comment:
- [ ] Data is not being displayed from the local storage, please check this video: 

https://user-images.githubusercontent.com/77038610/159342287-aa34aa55-c463-45b8-a7c9-22a28c81056d.mp4

- [ ] Make sure that after page refresh the all the data from localStorage is displayed

Reply:
@ShoiraTa  Last week, my partner (who owned this code) received an email saying that this requirement was going to be optional. Is there a way that can continue to be honored?
I feel this exercise today was more based on ES6 and using Luxon where this requirement feels like something that should have been made required last week. If we can negotiate that I would appreciate it. If not I will start working on it. Sorry to contest your requirement. Thank you!